README.md

```diff
@@ -59,10 +59,11 @@
 - (Optional) Install a third-party antivirus product of your choice.
 - Activate Windows using a valid license key. If you don't already have one, ask HR about getting one.
 - Install all available Microsoft updates, restarting your computer as necessary.
+- Open PowerShell with Administrator privileges. Run `Set-ExecutionPolicy RemoteSigned`. Type Y and press Enter to confirm that you want to make this change.
```

Comment:
@stephengtuggy, is this making Windows clients more secure or less?  See defaults below.

>AllSigned. Requires that all scripts and configuration files are signed by a trusted publisher, including scripts written on the local computer.

> Bypass. Nothing is blocked and there are no warnings or prompts.
Default. Sets the default execution policy. Restricted for Windows clients or RemoteSigned for Windows servers.

> RemoteSigned. Requires that all scripts and configuration files downloaded from the Internet are signed by a trusted publisher. The default execution policy for Windows server computers.

> Restricted. Doesn't load configuration files or run scripts. The default execution policy Windows client computers.

> Undefined. No execution policy is set for the scope. Removes an assigned execution policy from a scope that is not set by a Group Policy. If the execution policy in all scopes is Undefined, the effective execution policy is Restricted.

 > Unrestricted. Loads all configuration files and runs all scripts. If you run an unsigned script that was downloaded from the Internet, you are prompted for permission before it runs. The default execution policy for non-Windows computers and cannot be changed.

Reply:
@dunlavy Without changing this setting, you cannot run ANY PowerShell scripts. Period. Regardless of origin; whether the script is signed or not; or any other factor.

I choose RemoteSigned so that I can run my own scripts without having to sign them, while requiring that third-party scripts **are** signed before I run them.

Reply:
Yeah, this is a weird PowerShell permission thing. I always have to enable it when I do a new Windows set up.